### DISCUSSION TITLE
Please add redirect method navigate:true for backend component. Not for blade.php

### DISCUSSION THREAD

**Discussion by serhatkurt** [livewire/livewire#7308] [1. Help] [answered]
For Example;

php

namespace App\Livewire;

use Livewire\Component;

class CreatePost extends Component
{

    public function todoList()
    {
        //please add navigate() method

        return redirect()->route('posts')->navigate();
        
        or
        
        return redirect('/posts', navigate: true);
        
        beaceuse this method just : void

    }

    public function render()
    {
        return view('livewire.create-post');
    }
}

**Comment by joshhanley** [comment, marked as answer]
@serhatkurt you can already do that, see here https://livewire.laravel.com/docs/navigate#redirects

**Reply by serhatkurt** [reply]
Yes. I reviewed this in detail. And I wrote this request after the review.

If you try it in the backend, you will see that the method does not work.

This method just for the frontend.

**Reply by serhatkurt** [reply]
Just : void {}   method

**Reply by joshhanley** [reply]
@serhatkurt I don't understand what you mean about void method. Can you share your code so we can investigate? Thanks!

**Reply by serhatkurt** [reply]
namespace App\Livewire\Posts;

use Livewire\Attributes\Layout;
use Livewire\Component;

class PostCreate extends Component
{

public function comeBack(){

        return $this->redirect('/posts', navigate: true);
        
        // or 
                
        return redirect('/posts', navigate: true);
        
      
    }
    
}

**Reply by serhatkurt** [reply]
Void method is the method used only for the front end.

void codes are Livewire codes running in blade.php.

I'm talking about running it as navigate:true on the backend without using blade.php.
